app/ante/cosmos/authz_test.go

```diff
@@ -414,37 +417,46 @@ func (suite *AnteTestSuite) TestRejectMsgsInAuthz() {
 				tx  sdk.Tx
 				err error
 			)
+			nw := suite.GetNetwork()
```

Comment:
Do we have to call it in every iteration?

Reply:
yes, 'cause on every test the setup function runs to start fresh for each test case